hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -108,12 +110,12 @@ public class XceiverClientGrpc extends XceiverClientSpi {
    * Constructs a client that can communicate with the Container framework on
    * data nodes via DatanodeClientProtocol.
    *
-   * @param pipeline - Pipeline that defines the machines.
-   * @param config   -- Ozone Config
+   * @param pipeline     - Pipeline that defines the machines.
+   * @param config       -- Ozone Config
    * @param trustManager - a {@link ClientTrustManager} with proper CA handling.
    */
   public XceiverClientGrpc(Pipeline pipeline, ConfigurationSource config,
-      ClientTrustManager trustManager) {
+                           ClientTrustManager trustManager) {
```

Comment:
Please update your editor so that formatting changes are not made for lines you are not updating.

Reply:
done